docs/ReleaseHistory.md

```diff
@@ -10,6 +10,9 @@
 - FPS => False positive reduction in static analysis.
 - FNS => Flase negative reduction in static analysis.
 
+# 1.4.22 - 05/21/2024
```

Comment:
>05/21/2024 [](http://example.com/codeflow?start=11&length=10)

this should be UNRELEASED

Reply:
updated to 1.4.23, correct?

Reply:
Here's your entry. This is going into 1.4.23 but it didn't ship yesterday, it is unreleased. 

`1.4.23 - UNRELEASED`

Reply:
done